Tools/setup/arch.sh

```diff
@@ -69,8 +69,8 @@ sudo pacman -Sy --noconfirm --needed \
 
 # Python dependencies
 echo "Installing PX4 Python3 dependencies"
-pip install --upgrade pip setuptools wheel
-pip install -r ${DIR}/requirements.txt
+sudo pip install --upgrade pip setuptools wheel
```

Comment:
Actually, I don't think you should re-install pip like that. It's already taken care of by `pacman`. And do you really need `setuptools` and `wheel`? I would think that comes pre-installed.

Reply:
Good point, I took that over from the Ubuntu script, let me check.

Reply:
You were right, that line is totally unnecessary. I corrected including some comments.